packages/grid/_modules_/grid/components/columnHeaders/GridColumnHeaders.tsx

```diff
@@ -35,7 +36,7 @@ export const GridColumnsHeader = React.forwardRef<HTMLDivElement, {}>(function G
 
   const handleDragOver =
     !disableColumnReorder && apiRef
-      ? (event) => apiRef.current.onColHeaderDragOver(event, ref as React.RefObject<HTMLElement>)
+      ? (event) => apiRef.current.publishEvent(GRID_COLUMN_REORDER_DRAG_OVER_HEADER, event)
```

Comment:
missing `GridColumnHeaderParams`

Reply:
I can't pass `GridColumnHeaderParams` because I don't have the field here.

Reply:
you can retrieve it on the element target. We have `domUtils ` function to help